contribs/gmf/src/directives/layertree.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * @fileoverview This file provides the "layertree" directive for GeoMapFish
+ * applications.
+ *
+ * FIXME
+ */
+goog.provide('gmf.layertreeDirective');
+
+goog.require('gmf');
+goog.require('ngeo.LayertreenodeController');
+goog.require('ngeo.layertreeDirective');
+goog.require('ol.layer.Image');
+goog.require('ol.source.ImageWMS');
+
+
+/**
+ * @return {angular.Directive} The directive Definition Object.
+ * @ngInject
+ */
+gmf.layertreeDirective = function() {
+  return {
+    scope: {
+      'getMapFn': '&gmfLayertreeMap',
```

Comment:
I could have asked the question in your previous PR: does "getMapFn" stand for "getMap function"? If yes, what do you mean since in the HTML code "gmf-layertree-map" is a map object?